deploy/01_deploy_t.ts

```diff
@@ -0,0 +1,40 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+import { BigNumber } from "ethers"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { deployer } = await getNamedAccounts()
+  const { execute, read, log } = deployments
+
+  const T = await deployments.deploy("T", {
+    from: deployer,
+    log: true,
+  })
+
+  // We're minting 10B T, which is the value of the T supply on the production
+  // environment.
+
+  await execute(
+    "T",
+    { from: deployer },
+    "mint",
+    deployer,
+    BigNumber.from(10).pow(28)
+  )
```

Comment:
Can we extract minting to a separate step? There is a chance that on mainnet we'd like to hold on with minting and delegate this responsibility to DAO.

Reply:
Moved to a separate script, see 385e647864d2dfafa55ceff8beec17132f897fea.